opm/simulators/timestepping/AdaptiveTimeStepping_impl.hpp

```diff
@@ -242,8 +250,9 @@ namespace Opm {
             }
             catch (const Opm::NumericalProblem& e) {
                 substepReport += solver.failureReport();
+                const double newTimeStep = restart_factor_ * dt;
```

Comment:
This is wrong, as the substepTimer can adjust the new time step. Also, it is not necessary to do it here, as the logException call should be as before.